src/react/components/toolbar/toolbarItem.tsx

```diff
@@ -73,14 +74,20 @@ export abstract class ToolbarItem extends React.Component<IToolbarItemProps> {
             className.push("active");
         }
 
+        const accelerators = this.props.accelerators;
+
         return (
             <Fragment>
-                {this.props.accelerator &&
-                    <KeyboardBinding
-                        keyEventType={KeyEventType.KeyDown}
-                        accelerator={this.props.accelerator}
-                        onKeyEvent={this.onClick}
-                    />
+                {
```

Comment:
Instead making the accelerators on the toolbar item accept an array, lets move this down into the actual `KeyboardBinding` so anyone can pass 1 or more accelerators to a keyboard binding to have them mapped automatically.